generator/util/strFunctions.py

```diff
@@ -507,15 +507,25 @@ def wrap_token(name, pkg=''):
         return '\\token{' + pkg + ':\\-' + name + '}'
 
 
-def wrap_type(given_type, element, hack=False):
+def wrap_type(given_type, element='', hack=False):
     """
     Wraps the given type in a string that will be used in a latex
     description of that type.
 
     :param given_type: the type we want to wrap, e.g. 'array', 'enum',...
-    :param element:
+    :param element: where given_type is a collection this gives the
+        name of the element in the collection
+        OR
+        if we want the type of this element directly (hack = True)
```

Comment:
Maybe a better name for `hack` would be `get_element_type_directly`, which is more informative!